### PR TITLE
August news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,629 @@
...
+
+## rOpenSci HQ
+
+### Community call: Navigating the R ecosystem using R-universe on September 24!
+
+On [Tuesday, 24 September 2024 16:00 UTC](/commcalls/nov2024-r-universe/) (no RSVP needed), join us to learn more about R-Universe and how you can use it to improve your R package development workflow.
+
+In this community call, [Jeroen Ooms](/author/jeroen-ooms/) will provide details on what R-Universe is and an update on what you can do with it today. 
+He will also discuss the future of R-universe and how it can be used to navigate the R ecosystem.
+
```
Comment (yabellini) [inline content/blog/2024-08-19-news-august-24/index.Rmd:52]:
```suggestion
### [Webinar Screen Reader Accessible Tools and Resources for Learning and Working with R](/events/rsr-learn-and-use/)

[Liz Hare](https://ropensci.org/author/liz-hare/) and [Alican Cagri Gokcek](https://ropensci.org/author/alican-cagri-gokcek/), both participants in our [Champions Program](https://ropensci.org/champions/) will participate in a panel sharing their experiences with screen reader-accessible tools and resources for learning and working with R.

The event is co-organized by rOpenSci and the [Boğaziçi University](https://bogazici.edu.tr/en-US/Index). 
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,629 @@
...
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+ TODO STEFFI
+
+And remember, you can always cowork independently on work related to R, work on packages that tend to be neglected, or work on what ever you need to get done!
+
```
Comment (yabellini) [inline content/blog/2024-08-19-news-august-24/index.Rmd:76]: ### [Two years and twelve projects as Community Manager at rOpenSci](https://yabellini.netlify.app/blog/2024-06-23-ropensci-2years-cm-en/)

In June 2022 Yani became the community manager of rOpenSci. She started a series of blog post to share 12 projects she was involved in these two years to tell you more about the kind of work and activities a community manager of a technology community of practice does and what she learned in the process.

The series is also in Spanish.

Comment (yabellini) [inline content/blog/2024-08-19-news-august-24/index.Rmd:76]:
```suggestion
### [Two years and twelve projects as Community Manager at rOpenSci](https://yabellini.netlify.app/blog/2024-06-23-ropensci-2years-cm-en/)

In June 2022 Yani became the community manager of rOpenSci. She started a series of blog post to share 12 projects she was involved in these two years to tell you more about the kind of work and activities a community manager of a technology community of practice does and what she learned in the process.

The series is also in Spanish. 
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,652 @@
...
+In this community call, [Jeroen Ooms](/author/jeroen-ooms/) will provide details on what R-Universe is and an update on what you can do with it today. 
+He will also discuss the future of R-universe and how it can be used to navigate the R ecosystem.
+
+### Webinar: Screen Reader Accessible Tools and Resources for Learning and Working with R
+
+[Liz Hare](/author/liz-hare/) and [Alican Cagri Gokcek](/author/alican-cagri-gokcek/), both participants in our [Champions Program](/champions/) will participate in a [panel]((/events/rsr-learn-and-use/)) sharing their experiences with screen reader-accessible tools and resources for learning and working with R.
+
+The [event]((/events/rsr-learn-and-use/)) is co-organized by rOpenSci and the [Boğaziçi University](https://bogazici.edu.tr/en-US/Index) and will be held on **September 10**. 
+
+### Blog post series: two years and twelve projects as Community Manager at rOpenSci
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:59]:
```suggestion
### Blog post series: Two years and twelve projects as Community Manager at rOpenSci
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,652 @@
...
+
+In June 2022 Yani became the community manager of rOpenSci.
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:61]:
```suggestion
In June 2022 Yani became the Community Manager of rOpenSci. 
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,652 @@
...
+### Webinar: Screen Reader Accessible Tools and Resources for Learning and Working with R
+
+[Liz Hare](/author/liz-hare/) and [Alican Cagri Gokcek](/author/alican-cagri-gokcek/), both participants in our [Champions Program](/champions/) will participate in a [panel]((/events/rsr-learn-and-use/)) sharing their experiences with screen reader-accessible tools and resources for learning and working with R.
+
+The [event]((/events/rsr-learn-and-use/)) is co-organized by rOpenSci and the [Boğaziçi University](https://bogazici.edu.tr/en-US/Index) and will be held on **September 10**. 
+
+### Blog post series: two years and twelve projects as Community Manager at rOpenSci
+
+In June 2022 Yani became the community manager of rOpenSci. 
+She started a [series of blog posts](https://yabellini.netlify.app/blog/2024-06-23-ropensci-2years-cm-en/)) to share 12 projects she was involved in these two years to tell you more about the kind of work and activities a community manager of a technology community of practice does and what she learned in the process.
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:62]:
```suggestion
Now she has started a [series of blog posts](https://yabellini.netlify.app/blog/2024-06-23-ropensci-2years-cm-en/) to share 12 projects she was involved in these two years to tell you more about the kind of work and activities a community manager of a technology community of practice does and what she learned in the process.
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,652 @@
...
+The databases are published with [the v0.1.6 release of pkgstats](https://github.com/ropensci-review-tools/pkgstats/releases/tag/v0.1.6), and can be downloaded from there.
+Alternatively, to know how "noteworthy" your package is compared to CRAN packages, simply call [pkgcheck](https://docs.ropensci.org/pkgcheck) on your package (perhaps with `goodpractice = FALSE` to speed things up by skipping those parts of checks). Then, either print the results directly in the console, or use `out <- checks_to_markdown(checks, render = TRUE)` to generate and automatically open a rendered HTML version, where "Statistical Properties" will include the comparison of your package to all current CRAN packages.
+
+### The rOpenSci community at upcoming events
+
+Meet rOpenSci team and community members at events in the near future!
+
+- rOpenSci software research scientist Mark Padgham will be at the [RSECon, the Research Software Engineering Conference 2024](https://rsecon24.society-rse.org/) in Newcastle, UK from 3rd to 5th September 2024 where he will deliver his talk ["rOpenSci Statistical Software Peer Review: The importance and challenge of community engagement"](/events/rseuk-2024/).
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:78]:
```suggestion
- rOpenSci Software Research Scientist [Mark Padgham](/author/mark-padgham/) will be at the [RSECon, the Research Software Engineering Conference 2024](https://rsecon24.society-rse.org/) in Newcastle, UK from 3rd to 5th September 2024 where he will deliver his talk ["rOpenSci Statistical Software Peer Review: The importance and challenge of community engagement"](/events/rseuk-2024/).
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,652 @@
...
+
+- rOpenSci director Noam Ross and rOpenSci Lead Infrastructure Engineer Jeroen Ooms will take part in the [NumFOCUS Project Summit 2024](/events/numfocussummit-2024/) on [September 5-7, 2024](https://numfocus.org/2024-project-summit-openmbee) in Cambridge, Massachusetts, USA. Come to chat if you are around!
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:80]:
```suggestion
- rOpenSci Director [Noam Ross](/author/noam-ross/) and rOpenSci Lead Infrastructure Engineer [Jeroen Ooms](/author/jeroen-ooms/) will take part in the [NumFOCUS Project Summit 2024](/events/numfocussummit-2024/) on [September 5-7, 2024](https://numfocus.org/2024-project-summit-openmbee) in Cambridge, Massachusetts, USA. Come to chat if you are around!
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,652 @@
...
+
+- rOpenSci community manager Yanina Bellini Saibene will deliver a keynote talk at [BioNT Community Event & CarpentryConnect-Heidelberg 2024](/events/carpentryconnect-2024/), on November 14th in Heidelberg, Germany.
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:82]:
```suggestion
- rOpenSci Community Manager [Yanina Bellini Saibene](/author/yanina-bellini-saibene/) will deliver a keynote talk at [BioNT Community Event & CarpentryConnect-Heidelberg 2024](/events/carpentryconnect-2024/), on November 14th in Heidelberg, Germany.
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,652 @@
...
+
+Read [all about coworking](/blog/2023/06/21/coworking/)!
+
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+ TODO STEFFI
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:94]: If I'm very lucky, I'll have a theme/cohost for October by Thurs/Friday (when are you publishing?). 

```suggestion
- ~~Tuesday, September 3rd, 9:00 Australia Western (01:00 UTC)~~ **Cancelled**
- Tuesday, October 1st, 14h00 Europe Central (12:00 UTC), Theme TBA with cohost TBA and [Steffi LaZerte](/author/steffi-lazerte).
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 10]
```diff
@@ -0,0 +1,652 @@
...
+
+Etienne Bacher created an enticing R package called [flint](https://flint.etiennebacher.com/), that finds _and fixes_ lints in R code.
+Imagine [lintr](https://lintr.r-lib.org/) being as active as [styler](https://styler.r-lib.org/) instead of just telling you what to amend. :grin:
+Note that at the moment, flint does not have as many rules as lintr.
+
+The existence of flint is yet another benefit from [Davis Vaughn's building an R grammar for tree-sitter](/blog/2024/07/19/news-july-2024/#on-tools), since flint builds on Etienne Bacher's [astgrepr](https://astgrepr.etiennebacher.com/), that binds the [Rust ast-grep crate](https://ast-grep.github.io/), that in turns... uses tree-sitter!
+
+### Create content for help pages on the fly
+
+You can create dynamic content for a manual page using `#' \Sexpr[results=rd,stage=render]{<some-code>}`.
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:633]: 🙀 
```suggestion
Did you know that you can create dynamic content for the help page of a function in your R package using `#' \Sexpr[results=rd,stage=render]{<some-code>}`?
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 11]
```diff
@@ -0,0 +1,652 @@
...
+The code can even call an internal function.
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:634]:
```suggestion
The code can even call an internal function!
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 12]
```diff
@@ -0,0 +1,652 @@
...
+[Minimal example](https://github.com/maelle/personalized.help).
+
+Thanks Rich FitzJohn for sharing about this idea that he uses in his [stevedore package](https://github.com/cran/stevedore/blob/74b63bfcff2af200367fbd78febfd2d36855d500/R/help.R#L4).
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:637]:
```suggestion
Thanks Rich FitzJohn for sharing about idea that he uses in his [stevedore package](https://github.com/cran/stevedore/blob/74b63bfcff2af200367fbd78febfd2d36855d500/R/help.R#L4).
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 13]
```diff
@@ -0,0 +1,652 @@
...
+
+Relatedly, if you want to provide different content in the manual page depending on the OS, that's [possible](https://cran.r-project.org/doc/manuals/R-exts.html#Platform_002dspecific-sections).
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:639]:
```suggestion
Relatedly, if you want to provide different content in the manual page depending on the OS, that's also [possible](https://cran.r-project.org/doc/manuals/R-exts.html#Platform_002dspecific-sections).
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 14]
```diff
@@ -0,0 +1,652 @@
...
+
+If you're taking a bit further and what to change what `?thing` returns, you might be interested in these two strategies (not necessarily CRAN-compatible!):
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:641]:
```suggestion
If you're taking it a bit further and want to change what `?foo` returns, you might be interested in these two strategies (but be warned, these are not necessarily CRAN-compatible!):
```

File: content/blog/2024-08-19-news-august-24/index.Rmd [hunk 15]
```diff
@@ -0,0 +1,652 @@
...
+
+- How Elio Campitelli's rhelpi18n [currently overwrites the `.getHelpFile()`](https://github.com/eliocamp/rhelpi18n/blob/main/R/zzzz.R) function to make it possible to get a manual page in the correct language.
```
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:643]:
```suggestion
- Elio Campitelli's rhelpi18n package [currently overwrites the `.getHelpFile()`](https://github.com/eliocamp/rhelpi18n/blob/main/R/zzzz.R) function to make it possible to get a manual page in the correct language.
```

File: content/blog/2024-08-19-news-august-24/index.Rmd
Comment (steffilazerte) [inline content/blog/2024-08-19-news-august-24/index.Rmd:1]: Adjust date in folder name?

**Review by steffilazerte — COMMENTED** [review]
Looks good @maelle. Do you need to adjust the dates in the folder names?

* [ ] publication date is ok
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

File: content/blog/2024-08-19-news-august-24/index.Rmd
Comment (maelle) [reply content/blog/2024-08-19-news-august-24/index.Rmd:1]: Done, thank you!

**Comment by maelle** [comment]
Thanks a lot @steffilazerte!

**Pull request merged**